M2/Macaulay2/m2/packages.m2

```diff
@@ -224,6 +225,7 @@ newPackage = method(
 	UseCachedExampleOutput    => null,
 	Version                   => "0.0"
 	})
+newPackage Sequence := opts -> x -> newPackage(x#0, opts) -- to allow null entries
```

Comment:
Would `newPackage(String, Nothing)` make sense instead to solve the specific issue of trailing commas?

With the new behavior, I'd be a little concerned that something like `newPackage("Foo", 1, 2, 3, 4, 5)` would be allowed.

Reply:
I feel like `methods newPackage` showing `(newPackage, Sequence)` is a lot less surprising than `(newPackage, String, Nothing)`. I can assert that the other entries are null if you want?

Reply:
Sure, that works